### PR TITLE
No internet connection message in sidebox

### PR DESCRIPTION
**Pull request by Kusum-Dhoundiyal**
### What do you want to achieve?
- No internet connection message in sidebox

### PR Checklist

- [x] I have tested it locally and all functionalities are working fine.
- [ ] I have compared it with mocks and all design elements are the same.
- [ ] I have tested it in IE Browser.

### How was the code tested?

-
Screenshot 2020-07-20 at 1 44 17 PM

### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

NOTE: Make sure you're comparing your branch with the correct base branch

### CODE REVIEW

File: webplugin/template/mck-sidebox.html [hunk 1]
```diff
@@ -57,8 +57,9 @@ <h4 id="mck-conversation-title" class="mck-box-title mck-truncate">Conversations
 					</div>
 
 				</div>
-				
 			</div>
+			<div id="internet-disconnect-msg" class="km-offline-msg n-vis"> 
+				Uh oh!Looks like you're not connected to the internet, please check your network settings and try again. </div>
```
Comment (devashishmamgain) [inline webplugin/template/mck-sidebox.html:62]: move it to MCK_LABELS

File: webplugin/js/app/labels/default-labels.js [hunk 2]
```diff
@@ -92,6 +92,7 @@ Kommunicate.defaultLabels = {
     'how.to.reachout': 'How may we reach you?',
     'email.error.alert': 'It seems you have entered an invalid email',
     'conversation.rated':'rated the conversation',
+    'offline.msg': "Uh oh!Looks like you're not connected to the internet, please check your network settings and try again.",
```
Comment (devashishmamgain) [inline webplugin/js/app/labels/default-labels.js:95]: add space before Looks

**Comment by Kusum-Dhoundiyal** [comment]
> @Kusum-Dhoundiyal can you confirm color guidelines from Sriti for ADA complaince

@devashishmamgain  This color code is given by sriti only.

File: webplugin/js/app/mck-sidebox-1.0.js [hunk 3]
```diff
@@ -2117,6 +2119,7 @@ var KM_ATTACHMENT_V2_SUPPORTED_MIME_TYPES = ["application","text","image"];
             };
             _this.setLabels = function () {
                 $applozic('#mck-conversation-title').html(MCK_LABELS['conversations.title']).attr('title', MCK_LABELS['conversations.title']);
+                $applozic('#internet-disconnect-msg').html(MCK_LABELS['offline.msg']).attr('title', MCK_LABELS['offline.msg']);
```
Comment (ab14bhardwaj) [inline webplugin/js/app/mck-sidebox-1.0.js:2122]: @Kusum-Dhoundiyal Use Javascript here.

File: webplugin/template/mck-sidebox.html [hunk 4]
```diff
@@ -57,8 +57,8 @@ <h4 id="mck-conversation-title" class="mck-box-title mck-truncate">Conversations
 					</div>
 
 				</div>
-				
 			</div>
+			<div id="internet-disconnect-msg" class="km-offline-msg n-vis">  </div>
```
Comment (ab14bhardwaj) [inline webplugin/template/mck-sidebox.html:61]: @Kusum-Dhoundiyal "km-internet-disconnect-msg" follow this convention only.

**Pull request merged**